src/Telnyx.net/Entities/Calls/CallControlApplications/CallControlApplicationInbound.cs

```diff
@@ -0,0 +1,48 @@
+namespace Telnyx.net.Entities.Calls.CallControlApplications
+{
+    using System.Runtime.Serialization;
+    using Newtonsoft.Json;
+
+    public class CallControlApplicationInbound
+    {
+        /// <summary>
+        /// When set, this will limit the total number of inbound calls to phone numbers associated with this connection.
+        /// </summary>
+        /// <value>When set, this will limit the total number of inbound calls to phone numbers associated with this connection.</value>
+        [JsonProperty("channel_limit")]
+        public int? ChannelLimit { get; set; }
+
+        /// <summary>
+        /// Specifies a subdomain that can be used to receive Inbound calls to a Connection, in the same way a phone number is used, from a SIP endpoint. Example: the subdomain \&quot;example.sip.telnyx.com\&quot; can be called from any SIP endpoint by using the SIP URI \&quot;sip:@example.sip.telnyx.com\&quot; where the user part can be any alphanumeric value. Please note TLS encrypted calls are not allowed for subdomain calls.
+        /// </summary>
+        /// <value>Specifies a subdomain that can be used to receive Inbound calls to a Connection, in the same way a phone number is used, from a SIP endpoint. Example: the subdomain \&quot;example.sip.telnyx.com\&quot; can be called from any SIP endpoint by using the SIP URI \&quot;sip:@example.sip.telnyx.com\&quot; where the user part can be any alphanumeric value. Please note TLS encrypted calls are not allowed for subdomain calls.</value>
```

Comment:
Side note: <value> doesnt actually work as a comment. Can move to Summary section

Reply:
done.